eslint/src/index.ts

```diff
@@ -43,7 +24,12 @@ export const configs = {
   },
   'default-5': {
     rules: {
-      ...buildDefault(DEPRECATED_4_RULES, 'error')
+      ...buildDefault(DEPRECATED_5_RULES, 'warn')
```

Comment:
here and below is ‚5’, but what about ‚4’?

Reply:
We are going to release 5.0.0 so 4 leave in past

Reply:
oh, it’s ‚default-5’, now I see it